build/ci/community/prepare_env.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -xel
+set +o pipefail
+operator_package_file_path=deploy/olm-catalog/ibm-block-csi-operator-community/ibm-block-csi-operator.package.yaml
+csv_version=`yq eval .channels[0].currentCSV $operator_package_file_path`
```

Comment:
will this test non-stable channels? e.g. 1.5.1

Reply:
no, we would want that it will be tested in the ci? because its a one time thing

Reply:
I guess we need to find some way to test all non GA versions, but since versions like 1.5.1 are not common, I agree it's less important.
could you open a ticket for it and write its number here?

Reply:
done
CSI-3171